editor/index.html

```diff
@@ -6,6 +6,10 @@
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<!-- Origin Trial Token, feature = WebVR (For Chrome M62+), origin = https://threejs.org, expires = 2018-05-12 -->
 		<meta http-equiv="origin-trial" data-feature="WebVR (For Chrome M62+)" data-expires="2018-05-12" content="AkM+gi6Fb7IUBwFLs18hn/x6dusbsVRYG9KBTWUUjvmT0m0wGkxRPy63Aj/i+Ti4d6qMEMnRcsR7MVV5Wuvp/AIAAABQeyJvcmlnaW4iOiJodHRwczovL3RocmVlanMub3JnOjQ0MyIsImZlYXR1cmUiOiJXZWJWUjEuMU02MiIsImV4cGlyeSI6MTUyNjExODIwMH0=">
```

Comment:
there's [a new WebVR Origin Trial](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/available-trials.md#current-experimental-features) that ends on September 11, 2018. let me know if you want me to file a follow-up issue to track this.

Reply:
see #14023. The PR renewed the existing tokens. I received an email about the new tokens (extended WebVR support through Chrome M68) but i'm not sure about the order. I still see only this selection:

![image](https://user-images.githubusercontent.com/12612165/39914553-a1aa124e-5505-11e8-8678-0edf167b1a66.png)

Do I have to choose the first one?